bot/exts/evergreen/pythonfacts.py

```diff
@@ -0,0 +1,28 @@
+import random
+
+import discord
+from discord.ext import commands
+
+
+with open('bot/resources/evergreen/python_facts.txt') as file:
+    FACTS = list(file)
+
+COLORS = [0x4B8BBE, 0xFFD43B, ]
```

Comment:
Get the colours from `constants.py` or add this there.

Reply:
Done